### PR TITLE
Add reden_update_check

### PR DESCRIPTION
**Pull request by ikunsz666**
---

## 检查以下事项 | Check the following:

- 仓库 URL、分支和 `related_path` 正确且 `mcdreforged.plugin.json` 位于应当存在的位置
  Repository URL, branch and `related_path` are correct and `mcdreforged.plugin.json` file exists at the correct path

- 根据 [文档](https://mcdreforged.readthedocs.io/zh_CN/latest/plugin_dev/plugin_catalogue.html#label)，检查插件标签正确且合适  
  Plugin labels are correct and appropriate according to [documentation](https://mcdreforged.readthedocs.io/en/latest/plugin_dev/plugin_catalogue.html#label)

- 确保 `mcdreforged.plugin.json` 和 `plugin_info.json` 中的插件 ID 与在本仓库创建的文件夹名称一致
  Plugin ID in `mcdreforged.plugin.json` and `plugin_info.json` match the name of the folder created in this repository

- 在介绍中简要阐明插件功能，在 README 中详细阐明插件用法，避免含糊不清。确保潜在用户有理由考虑使用您的插件
  Briefly clarify plugin features in the introduction, and clarify plugin usage in detail in the README. Avoid vague descriptions. Ensure that potential users have a reason to consider using your plugin

- [x] 我已检查并确认以上所有项目
  I have checked and confirmed all the above

### CODE REVIEW

**Review by alex3236 — CHANGES_REQUESTED** [review]
plugin check failed;
missed `en_us` introduction

**Comment by alex3236** [comment]
另外，我不认为用 MCDR 管理模组是个好主意。目前已经有不少模组可以从 CurseForge 和 Modrinth
 检查和下载更新

**Comment by AnzhiZhang** [comment]
为什么仓库是 githubfast.com，存在部分地区无法访问的问题
image

**Comment by ikunsz666** [comment]
我反而是正常github无法连接所以才使用的fast，这个网站疑似会把所有包含github的网址全部替换为githubfast，我稍后用下魔法直接连github把它改回来

File: plugins/reden_update_check/plugin_info.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "id": "reden_update_check",
+    "authors": [
+        {
+            "name": "Hikari",
+            "link": "https://githubfast.com/ikunsz666"
+        }
+    ],
+    "repository": "https://githubfast.com/ikunsz666/reden_update_check",
+    "branch": "master",
+    "related_path": "reden_update_check",
```
Comment (AnzhiZhang) [inline plugins/reden_update_check/plugin_info.json:11]: https://mcdreforged.readthedocs.io/zh_CN/latest/plugin_dev/plugin_catalogue.html#related-path

**Review by AnzhiZhang — CHANGES_REQUESTED** [review]

**Comment by ikunsz666** [comment]
仓库文件结构已修改

**Review by AnzhiZhang — APPROVED** [review]

**Pull request closed without merging**